packages/fxa-oauth-console/app/adapters/application.js

```diff
@@ -129,16 +128,24 @@ export default DS.RESTAdapter.extend({
    * @param record
    * @returns {Array}
    */
-  buildURL: function(type, id, record) {
+  buildURL: function(type, id, record, requestType) {
     var url = [];
     var host = this.host;
     var prefix = this.urlPrefix();
 
-    // FxA OAuth API requires singular 'client' when the record id is set
-    if (record) {
+    switch (requestType) {
```

Comment:
This seems to be the `Ember way` of doing things: https://github.com/emberjs/data/blob/v3.8.0/addon/-private/adapters/build-url-mixin.js#L56